src/fn/fn-shell.scss

```diff
@@ -0,0 +1,85 @@
+@import "./fn-settings";
+@import "./fn-mixins";
+
+$block: #{$fn-namespace}-shell;
+
+.#{$block} {
+  @include fn-reset();
+
+  $fd-shell-action-buttons-padding: 0.4375rem;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0.75rem 1.5rem;
+  background: #ffffff;
+  height: 3.5rem;
+
+  &__group {
+    @include fn-reset();
+
+    flex-grow: 1;
+    flex-shrink: 1;
+    flex-basis: 0;
+
+    &--product {
+      align-items: center;
+      display: flex;
+      justify-content: flex-start;
+      order: 1;
```

Comment:
Why do you need order?

Reply:
To lay out these elements in the correct order L>R. I guess this can be removed and we can leave it up to the developer

Reply:
This was to encourage the dev to place these elements in the correct spot in the shellbar, but yes that might be too opinionated, i'll remove this